torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from typing import Any
+from torch.fx.node import Node
+
+def get_all_users_of(fx_module: GraphModule, index: int) -> List[int]:
+    """Given the graph(fx_module) and an index, return a list of all node indexes that use this node"""
+    """find_use is used to find out if the node is another node's arg or kwargs."""
+    def find_use(arg: Any) -> bool:
```

Comment:
Can you refactor this method out of get_all_users_of ? It might be useful elsewhere and then you can pass in two nodes as opposed to using a global current_node.